gwf/cli/status.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import absolute_import, print_function
+from .arg_parsing import SubCommand
+from ..utils import dfs
+from ..exceptions import TargetDoesNotExistsError
+from colorama import Fore, Back, Style
+import os
+from math import ceil
+
+
+__doc__ = """
```

Comment:
Doc string should just go to the top of the module (then it will be automatically assigned to `__doc__`).